jest/async-storage-mock.js

```diff
@@ -40,7 +40,7 @@ const asMock = {
   ),
 
   clear: jest.fn<[CallbackType], Promise<*>>(_clear),
-  getAllKeys: jest.fn<[], void>(),
+  getAllKeys: jest.fn<[], Promise<string[]>>(_getAllKeys),
```

Comment:
I am unsure if the types are correct on L43.

Reply:
Looks fine

Reply:
Thanks!